helix-front/tslint.json

```diff
@@ -12,7 +12,7 @@
     ],
     "curly": true,
     "eofline": true,
-    "forin": true,
+    "forin": { "severity": "off" },
```

Comment:
We can re-enable these rules which produced errors after upgrading our linting libraries later, in a future standalone PR.